pkg/pipeline/source/sdk/synchronizer.go

```diff
@@ -199,70 +221,101 @@ func (t *TrackSynchronizer) onSenderReport(pkt *rtcp.SenderReport, ntpStart time
 	}
 }
 
-func absGreater(value, max time.Duration) bool {
-	return value > max || value < -max
+func (t *TrackSynchronizer) getFrameSize() uint32 {
+	if frameSize := t.frameSize; frameSize != 0 {
+		return frameSize
+	}
+	return uint32(float64(t.clockRate) / 2.4)
```

Comment:
why 2.4? Would be good to have some comment in code explaining that.

Reply:
that was supposed to be 24, good catch!